.reek

```diff
@@ -22,17 +26,26 @@ TooManyMethods:
 UncommunicativeMethodName:
   exclude:
     - Users::PhoneConfirmationController
+    - render_401
 UnusedPrivateMethod:
   exclude:
     - ApplicationController
     - ActiveJob::Logging::LogSubscriber
+    - SamlIdpController
     - Users::PhoneConfirmationController
     - UpdateUserProfileForm
 UtilityFunction:
   public_methods_only: true
   exclude:
     - AnalyticsEventJob#perform
     - SmsSenderConfirmationJob
+
+'spec':
```

Comment:
What does this mean?   if it means what i think it means shouldn't `spec` be removed from the last line of this file?